cli/src/commands/dsl/validate.ts

```diff
@@ -44,11 +116,14 @@ export default class Validate extends BaseCommand {
       }
     );
 
-    const program = ts.createProgram(files, DEFAULT_COMPILER_OPTIONS);
+    const TSConfig = (await this.getTSConfig()) || DEFAULT_COMPILER_OPTIONS;
```

Comment:
We should probably log which ts config we're using

Reply:
It looks like we are logging if we use one from the env but in the case that we're not we should also log that